modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsForm.es.js

```diff
@@ -96,6 +97,12 @@ class ResultRankingsForm extends Component {
 		 */
 		dataMap: {},
 
+		/**
+		 * Disable buttons, checkboxes and text inputs if the status is not applicable.
+		 * @type {boolean}
+		 */
+		disabled: this.props.initialStatus === STATUS_TYPES.NOT_APPLICABLE,
+
```

Comment:
Since there's already a state for `status`, I think it's fine to remove this extra state `disabled` and just evaluate inline on the different components (saves us from keeping track of different states tied to one prop, even though it does make sense to evaluate disabled upfront).

So for Alias and the rest of the components with `disabled` prop, we can just do this inline check instead:
```

<Alias
	disabled={status === STATUS_TYPES.NOT_APPLICABLE}
	keywords={aliases}
	onChange={this._handleUpdateAliases}
/>
```